src/main/java/com/pif/insatplatform/service/dto/IntegrationDTO.java

```diff
@@ -0,0 +1,136 @@
+package com.pif.insatplatform.service.dto;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * A DTO for the Integration entity.
+ */
+public class IntegrationDTO implements Serializable {
+
+    private Long id;
+
+    private String accessToken;
```

Comment:
Remove access token from the DTO, we should not be sending senstive data to the UI